tox.ini

```diff
@@ -17,3 +17,7 @@ deps = -r{toxinidir}/requirements/lint.txt
 changedir = src
 commands = flake8 croniter/croniter.py
 
+[testenv:format]
+deps = -r{toxinidir}/requirements/format.txt
+changedir = src
+commands = black src
```

Comment:
as this is during ci, we certainly need to add `--check` flag areant we ?

Reply:
Ah yes, sorry!

Reply:
Should we then have two commands, `check-format` and `format`?

Reply:
Sounds good. I'll remove the `format` command then